components/view/navbar/Navbar.php

```diff
@@ -0,0 +1,94 @@
+<?php
+
+
+namespace app\components\view\navbar;
+
+use yii\base\InvalidConfigException;
+
+/**
+ * Represents the header navbar
+ * @package app\components\view\navbar
+ */
+class Navbar
+{
+    /**
+     * @var NavbarMainItem[] $items The navbar items
+     */
+    private $items;
+
+    /**
+     * Navbar constructor
+     *
+     * @param array $config The array representing the Navbar config
+     *  Format:
+     *  [
+     *      'items' => [
+     *          [
+     *              'title'         => 'Item title',
+     *              'url'           => 'URL when item clicked',
+     *              'roleNeeded'    => 'role'   // Optionnal, if not set everyone will see it
+     *              'activeActions' => [        // Optionnal, if not set the item will never be shown as active
+     *                  'controller-name'   => [
+     *                      'action-name',
+     *                      'action-name',
+     *                      // OR
+     *                      '*'
```

Comment:
This should be an array OR a '*' but not both.

Reply:
Well regarded !